src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -26,6 +26,8 @@ import UpdateUserCommentModal from 'src/pages/workspaces/workspace/jobHistory/Up
 import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
 
 
+const WORKFLOWS_TABLE_ROW_HEIGHT = FLEX_TABLE_DEFAULT_ROW_HEIGHT
```

Comment:
I know all-upper-case names are used frequently in various languages to assign constants to but we typically don't do that in this repo. For consistency, I'd suggest naming them `workflowsTableRowHeight` and `flexTableDefaultRowHeight`.

Reply:
Done in [db78a76](https://github.com/DataBiosphere/terra-ui/pull/2845/commits/db78a7626443352644c5ab36c4b7f6c7dc3437bf).